.pylintrc

```diff
@@ -0,0 +1,292 @@
+[MASTER]
```

Comment:
Was this generated using edx-pylint?  Isn't there usually a disclaimer at the top if that's the case?